app/javascript/packs/ajax.js

```diff
@@ -18,6 +18,12 @@ global.ajaxCall = (
     $("#browse-outlet a").each(function () {
       const link = $(this);
       const href = link.attr("href");
+      if (href.startsWith("mailto:")){
+        return;
+      }
+      if (href.startsWith("tel:")){
+        return;
+      }
```

Comment:
Maybe shrinking it down a bit would be better:

```suggestion
      if (href.startsWith("mailto:") || href.startsWith("tel:")){
        return;
      }
```

Or even something like:
```suggestion
      if (href.match(/(mailto|tel):(.*)/)) {
        return;
      }
```